recipes/codingorf/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "v1.0.0" %}
+{% set sha256 = "09dfe82f8b3d5572d302ab6ce6b87c1ad0f6a281b33163f4870150b126c2ff4e" %}
+
+package:
+  name: codingorf
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - ncbi-genome-download=ncbi_genome_download.__main__:main
```

Comment:
The entry points seems not to belong to this package

Reply:
When I learn how to write meta.yaml, I made a mistake.
I will take a look at and fix it.
Thank you!